doc/release-notes.md

```diff
@@ -75,19 +75,25 @@ There is a checkbox to mark the transaction as final.
 The RPC default remains unchanged: to use RBF, launch with `-walletrbf=1` or
 use the `replaceable` argument for individual transactions.
 
-Custom wallet directories
----------------------
-The ability to specify a directory other than the default data directory in which to store
-wallets has been added. An existing directory can be specified using the `-walletdir=<dir>`
-argument. Wallets loaded via `-wallet` arguments must be in this wallet directory. Care should be taken
-when choosing a wallet directory location, as if it becomes unavailable during operation,
-funds may be lost.
-
-Default wallet directory change
---------------------------
-On new installations (if the data directory doesn't exist), wallets will now be stored in a
-new `wallets/` subdirectory inside the data directory. If this `wallets/` subdirectory
-doesn't exist (i.e. on existing nodes), the current datadir root is used instead, as it was.
+Wallets directory configuration (`-walletdir`)
+----------------------------------------------
+
+Bitcoin Core now has more flexibility in where the wallets directory can be located.
+
+- For new installations (where the data directory doesn't already exist),
```

Comment:
I would add something to ground this in existing behavior. Maybe say "Previously wallet database files were stored at the top level of the bitcoin data directory. Now for new installations [...]"

Reply:
Good suggestion. Added.